tests/FSharp.Data.GraphQL.IntegrationTests.Server/Schema.fs

```diff
@@ -46,7 +45,7 @@ module Schema =
                   Define.Input("stringOption", Nullable StringType, description = "A string option value.")
                   Define.Input("intOption", Nullable IntType, description = "An integer option value.")
                   Define.Input("uri", UriType, description = "An URI value.")
-                  Define.Input("guid", GuidType, description = "A Guid value.") ])
+                  Define.Input("guid", IDType, description = "A Guid value.") ])
```

Comment:
I don't think it should be `ID` type. If we want to test it it must be as a separate item